index.html

```diff
@@ -0,0 +1,537 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Smashing Mag Clone</title>
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/styles.css" />
+  </head>
+
+  <body>
+
+    <!-- Header -->
+    <header class="header flex">
+      <nav class="nav flex">
+        <a href="index.html" class="logo">
+          <img
+            src="https://d33wubrfki0l68.cloudfront.net/81fa290218cedc39d32be7b1f1aba58a25829ebe/0a05f/images/logo--full.svg"
+            alt="Smashing Mag logo"
+          />
+        </a>
+        <ul class="nav-links flex">
+          <li><a href="#">Articles</a></li>
+          <li><a href="#">Guides</a></li>
+          <li><a href="#">Books</a></li>
+          <li><a href="#">Events</a></li>
+          <li><a href="#">Membership</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li>
+            <a class="menu-link" href="#"
+              >Menu &nbsp;<img
+                style="width: 20px;"
+                src="https://d33wubrfki0l68.cloudfront.net/e3f5fe5fb4ec15257cc4137f13a0d5081b4ea5f8/aca41/images/icons/menu-cat.svg"
+                alt="cat"
+            /></a>
```

Comment:
- [ ] For good coding practices, avoid using inline styling in your HTML, please add your styles in your CSS file